pyproject.toml

```diff
@@ -300,12 +275,14 @@ ignore = [
     "E501", "TCH001", "TCH002", "TCH003",
     "A003", # flake8-builtins - we have class attributes violating these rule
     "COM812", # flake8-commas - we don't like adding comma on single line of arguments
+    "COM819", # conflicts with ruff-format
     "SIM117", # flake8-simplify - we some of merged with statements are not looking great with black, reanble after drop python 3.9
     "RET504", # not fixed yet https://github.com/charliermarsh/ruff/issues/2950
```

Comment:
that issue was closed as implemented in a PR, worth revisiting?

Reply:
Maybe yes, but in separate PR.

Reply:
@psobolewskiPhD could you link to that PR?

Reply:
https://github.com/astral-sh/ruff/pull/3393
but there were two related followups
https://github.com/astral-sh/ruff/pull/4997
https://github.com/astral-sh/ruff/pull/4998